hack/jenkins/e2e-runner.sh

```diff
@@ -137,6 +137,14 @@ function install_google_cloud_sdk_tarball() {
     export PATH=${install_dir}/google-cloud-sdk/bin:${PATH}
 }
 
+function dump_cluster_logs_and_exit() {
+    local -r exit_status=$1
```

Comment:
Nit: might be worth just making this equal to `$?` and getting rid of the argument.